src/main/io/displayport_msp_dji_compat.c

```diff
@@ -260,9 +263,9 @@ uint8_t getDJICharacter(uint8_t ch, uint8_t page)
 /*
         case SYM_NM:
             return DJI_SYM_NM;
-*/
```

Comment:
I had to comment here for a few lines above. I believe symbols 0x7D and 0x7E are Km and Miles respectively.